webapp-django/crashstats/crashstats/views.py

```diff
@@ -136,6 +140,68 @@ def build_data_object_for_crash_reports(response_items):
     return crash_reports
 
 
+def get_report_list_from_super_search(**kwargs):
+    '''Return results as expected by the report_list views, but loaded
+    from Super Search.
+
+    This function converts given parameters to match what Super Search expects.
+    It changes some values (like version and _sort parameters). It then
+    queries Super Search, and maps the field names in the results to what
+    the report/list/ views and templates expect.
+    '''
```

Comment:
What's with these `'''` comments?! :)
you're the only one doing them. I guess it's your personal touch to say "adrian was here" :)